samples/net/full_mqtt_tls/Kconfig

```diff
@@ -0,0 +1,35 @@
+# Kconfig - Private config options for Google Cloud IOT application
+
+#
+# Copyright (c) 2018 Linaro
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+mainmenu "Google IOT MQTT/TLS sample application"
+
+config CLOUD_CLIENT_ID
+	string "Client ID string"
+	default "put client string here"
+	help
+	  Set required client string that matches Google Cloud project
+
+config CLOUD_AUDIENCE
+	string "Audience string"
+	default "put audience string here"
+	help
+	  Set required audience string that matches Google Cloud project
+
+config CLOUD_SUBSCRIPTION
+	string "Subscription string"
+	default "put subscription string here"
+	help
+	  Set required subscription string that matches Google Cloud project
+
+config CLOUD_PUBLISH_TOPIC
+	string "Publish topic string"
+	default "put topic string here"
+	help
+	  Set publish topics string that matches Google Cloud project
+
+source "$(ZEPHYR_BASE)/Kconfig.zephyr"
```

Comment:
Can make this just

    source "Kconfig.zephyr"

See https://github.com/zephyrproject-rtos/zephyr/pull/10487.

Reply:
done